### ISSUE TITLE
frameUncons not exported

### ISSUE THREAD

**Issue by dalejordan** [acowley/Frames#35]
Did you mean to leave this function off the export list?

It seems to be more general than recUncons which is exported.

The naming of frameCons, frameConsA, frameUncons and frameSnoc seems confusing since they operate at the Rec level and have nothing to do with Frames (the type, not the package).

**Comment by acowley** [comment]
You're right. I think the bad naming came from the overlap with Vinyl. Using the prefix `rec` is potentially confusing, but I'm not sure what would be better. We should figure out how to improve things, and do a deprecation cycle.
